analyzers/its/expected/Net5/recordsvsclasses-{96F1111E-2713-4CDE-B2D8-BF973EB5A4DE}-net5.0-S3353.json

```diff
@@ -2,7 +2,7 @@
 "issues":  [
 {
 "id":  "S3353",
-"message":  "Add the 'const' modifier to 'user'.",
+"message":  "Add the 'const' modifier to 'user', and replace 'var' with 'string?'.",
```

Comment:
This is a bit unfortunate but it compiles and is not wrong. I added a test to document it.